Datadog.Trace.sln

```diff
@@ -262,6 +262,16 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "UpdateVendors", "tools\Upda
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SynchronizeInstaller", "tools\SynchronizeInstaller\SynchronizeInstaller.csproj", "{EBDDF6D7-59B8-4831-80C6-458C1ABBFC63}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "demo", "demo", "{34015B45-43F6-4E29-BDC6-4E16CFAD79BE}"
+EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "AutomaticTraceIdInjection", "AutomaticTraceIdInjection", "{5DE795C0-3DB6-4DD6-9EAF-F6A6698AD712}"
```

Comment:
Thoughts on having these projects live in a separate solution? My VS already crawls loading 54 projects, and these are not needed for development.

Reply:
Yeah that's a good idea, 100% in support of that

Reply:
I agree too.

Reply:
Resolved in eb1bce01779b16c74be7bd02e5ab13dac0abea61